scripts/build-sign-release-images.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+: "${GIT_HASH:?Environment variable empty or not defined.}"
+: "${GIT_TAG:?Environment variable empty or not defined.}"
+: "${RUN_ID:?Environment variable empty or not defined.}"
+: "${RUN_ATTEMPT:?Environment variable empty or not defined.}"
+: "${REGISTRY:?Environment variable empty or not defined.}"
+
+echo "Signing images with Keyless..."
+digest=$(crane digest "${REGISTRY}":"${GIT_TAG}")
```

Comment:
It's a little unfortunate that we can't get the digest of the image directly from goreleaser. But this seems okay too.

Can we rename the file? Or just move it to `cosign sign` directly inside release.yaml?

Reply:
i will check when that will be ready in goreleaser :)